specs/networking/p2p-interface.md

```diff
@@ -231,23 +230,24 @@ Request/response messages MUST adhere to the encoding specified in the protocol
 
 ```
 request   ::= <encoding-dependent-header> | <encoded-payload>
-response  ::= <result> | <encoding-dependent-header> | <encoded-payload>
+response  ::= <response_chunk>+
+response_chunk  ::= <result> | <encoding-dependent-header> | <encoded-payload>
```

Comment:
Is the result-per-chunk pattern deliberate? Would the result be a global result per req/rep?